Updated Readme (#11)
pub-trivia/trivia-app: README.md

# Pub Trivia App

## Overview

> In these days of families and friends being spread out across wide geographical locations, and even more in our current coronavirus crisis, it is sometimes hard to be able to get together to share some fun. This is where our app comes in. We are creating a virtual pub quiz (bring your own beer!) where you can login and invite friends and family to join you for a quiz! You play against each other in real time, and watch your avatars race each other to be the winner. After the quiz your friends might want to sign up too and create a group, so you can keep competing again and again. When you're done playing why not contribute a question yourself to bamboozle someone else. 

Reviewer: @dropcat13 this is great! One small tweak, I think we decided the group thing would be a stretch goal so maybe we don't want to include this sentence:

> After the quiz your friends might want to sign up too and create a group, so you can keep competing again and again.

## Research
### Real Value to People
Not all families and friends live near each other. Family members move away for work or school, or friends who meet at college disperse around the world after they have graduated, but they still want to keep in contact and do something together. Cat's family lives in the USA, UK, Canada and Ireland and love to do quizzes... but there has to be a better way than sending telegram group messages!  
(screenshot here)

### Other similar apps

> * Jackbox games - paid for games that you can invite other to play with you, lots of little animations, and very fun to play. 

Reviewer: ```suggestion
* [Jackbox games](https://www.jackboxgames.com/) - paid for games that you can invite others to play with you, lots of little animations, and very fun to play. 
```

> * Quizlet - designed for studing with quizzes for math, history, languages etc. You can also create your own quizzes to suit your own study needs. 

Reviewer: ```suggestion
* [Quizlet](https://quizlet.com/) - designed for studying with quizzes for math, history, languages etc. You can also create your own quizzes to suit your own study needs. 
```

> * QuizUp - IOS app, you can select a catogory and engage in head to head battles with other random users. 

Reviewer: ```suggestion
* [QuizUp](https://www.quizup.com/en) - IOS app, you can select a category and engage in head to head battles with other random users. 
```

* Lots of websites where you can take all sorts of quizzes yourself, without competing with anyone else.  

These other apps and websites mainly give you the question with no ability to add your own, and no recurring group aspect. 

## UX/UI Flow
(screenshot here)

## Data Flow
(screenshot here)

## Technologies and Libraries used

* React
* MongoDB

> * 

Reviewer: ```suggestion
```

> * Websockets for synchronous playing

Reviewer: ```suggestion
* [socket.io](https://socket.io/) for synchronous playing
```

* Deployed using Heroku

## Team members and Roles

> Cythia Rich - Websockets and git guru

Reviewer: ```suggestion
- **Cynthia Rich** - integrating websockets for gameplay
```

> Catherine Kretzschmar - Quiz logic and gameplay

Reviewer: ```suggestion
- **Catherine Kretzschmar** - User authentication and quiz creation logic
```

> Mandy Torrence - frontend design and UX

Reviewer: ```suggestion
- **Mandy Torrence** - frontend design and UX
```

> Susan Marshall - backend (databases and models) 

Reviewer: ```suggestion
**Susan Marshall** - backend (databases and models), question creation and dashboard
```

## Project Management Board and Schedule
(screenshot here) 

## Acknowledgments

> Zack, Carl, Tim and Jon for their help during this course and for being the inspiration for our avatars. 

Reviewer: ```suggestion
Zach, Carl, Tim and Jon for their help during this course and for being the inspiration for our avatars. 
```
